### PR TITLE
Added bricks to smelting recipe

### PR DESCRIPTION
**Pull request by AlphaBuffalo**
As title states. Tested and it works, but with a weird bug that when you make bricks it also makes glass, I suspect it has something to do with the smelter itself or maybe something else but it shouldnt cause a problem since this is a minor content addition anyways.

### CODE REVIEW

**Comment by ghost** [comment]
>more things miners will never bother to smelt

**Comment by AlphaBuffalo** [comment]
I only added because wad was making a new station and was using the brick walls, I added brick walls before I added a smelting recipe for the bricks

File: code/modules/mining/smelting.dm [hunk 1]
```diff
@@ -80,6 +80,14 @@
 	)
 	yieldtype = /obj/item/stack/sheet/plasteel
 
+/datum/smelting_recipe/brick
+	name = "Brick"
+	ingredients=list(
+		MAT_IRON = 1,
+		MAT_SAND = 1
```
Comment (ComicIronic) [inline code/modules/mining/smelting.dm]: `MAT_GLASS`. This is the source of your bug.

**Review by ComicIronic — CHANGES_REQUESTED** [review]

**Comment by Shadowmech88** [comment]
Won't changing the materials to `MAT_IRON` and `MAT_GLASS` just make it require smelting an iron sheet and a glass sheet instead of smelting an iron sheet and a pile of sand?

**Comment by AlphaBuffalo** [comment]
/datum/smelting_recipe/glass
	name = "Glass"
	ingredients=list(
		MAT_GLASS = 1
	)
	yieldtype = /obj/item/stack/sheet/glass/glass

You need MAT_GLASS to make Glass

**Review by ComicIronic — APPROVED** [review]

**Comment by AlphaBuffalo** [comment]
Is this ready for merge?

**Comment by ghost** [comment]
Yes, just wait for c*llabs to do their god damn job

**Comment by Exxion** [comment]
>implying any of us have jobs

**Comment by Intigracy** [comment]
Did you test to make sure it works after the change?

**Comment by AlphaBuffalo** [comment]
it worked

**Pull request merged**